acp/common.php

```diff
@@ -41,97 +41,97 @@
 	'DIGESTS_HAS_UNSUBSCRIBED'								=> 'S’est désabonné',
-	'DIGESTS_HOUR_SENT'										=> 'Heure d’envoi (basée sur UTC%+d)',
+	'DIGESTS_HOUR_SENT'										=> 'Heure d’envoi (fondée sur UTC%+d)',
 	'DIGESTS_HOUR_SENT_GMT'									=> 'Heure d’envoi par défaut (UTC)',
 	'DIGESTS_IGNORE'										=> 'Ignorer les lignes cochées',
 	'DIGESTS_ILLOGICAL_DATE'								=> 'Vos date et heure sont invalides. Veuillez les corriger et les envoyer à nouveau en utilisant le format de date et heure : AAAA-MM-JJ HH:MM:SS.',
 	'DIGESTS_INCLUDE_ADMINS'								=> 'Inclure les administrateurs',
-	'DIGESTS_INCLUDE_ADMINS_EXPLAIN'						=> 'Permet d’abonner ou de désabonner les administrateurs en plus des utilisateurs normaux.',
+	'DIGESTS_INCLUDE_ADMINS_EXPLAIN'						=> 'Permet d’abonner ou de désabonner les administrateurs en complément des utilisateurs ordinaires.',
 	'DIGESTS_INCLUDE_FORUMS'								=> 'Toujours inclure les forums suivants',
-	'DIGESTS_INCLUDE_FORUMS_EXPLAIN'						=> 'Saisissez le forum_id des forums qui doivent figurer dans le résumé. Séparez les forum_id par des virgules. En indiquant 0, aucun forum ne sera inclus. Pour déterminer le forum_id d’un forum, observez la valeur du paramètre « f » dans le champ de l’URL. C’est le forum_id. Exemple: http://www.example.com/phpBB3/viewforum.php?f=1. Ne pas utiliser de forum_ids qui correspondent à des catégories. <i>Ce réglage est ignoré si l’option Sujets cochés seulement est cochée par l’utilisateur.</i>',
+	'DIGESTS_INCLUDE_FORUMS_EXPLAIN'						=> 'Saisissez le forum_id des forums qui doivent figurer dans le résumé. Séparez les forum_id par des virgules. En indiquant 0, aucun forum ne sera inclus. Le forum_id correspond à la valeur du paramètre « f » dans le champ de l’URL du forum. Exemple : http://www.example.com/phpBB3/viewforum.php?f=1. N’utilisez pas de forum_ids qui correspondent à des catégories. <i>Ce réglage est ignoré si l’option « Sujets cochés seulement » est activée par l’utilisateur.</i>',
 	'DIGESTS_LAST_SENT'										=> 'Dernier résumé envoyé',
 	'DIGESTS_LIST_USERS' 								=> array(
 																	1	=>	'%d Utilisateur',
 																	2	=>	'%d Utilisateurs',
 																),
-	'DIGESTS_LOWERCASE_DIGEST_TYPE'							=> 'Indiquer la fréquence d’envoi du résumé en lettres minuscules',
-	'DIGESTS_LOWERCASE_DIGEST_TYPE_EXPLAIN'					=> 'En Anglais, le titre du résumé sera par exemple « My board name Daily Digest ». Dans certaines langues « Digest Daily » précède le nom du forum. En indiquant Oui, la fréquence du résumé apparaîtra ainsi « Résumé journalier de mon forum », avec la première lettre de la fréquence d’envoi en minuscule.',
+	'DIGESTS_LOWERCASE_DIGEST_TYPE'							=> 'Indiquez la fréquence d’envoi pour le titre du résumé en lettres minuscules',
```

Comment:
Il sort d'où le `fréquence d’envoi` ?

En anglais il est dit : `Lowercase the digest type in digests`

Il faut qu'on trouve autre chose, car pour moi ca amène plus d'incompréhension qu'autre chose.

Reply:
La fréquence d'envoi, c'est le "digest type" qui est inclus dans le titre du courriel : 

<img width="944" alt="IMG_ 2022-10-16 at 16 12 57" src="https://user-images.githubusercontent.com/87082648/196040342-571cb6fc-48d8-42b8-92a2-a98a6cb70f67.png">

C’est d'ailleurs ce qui est expliqué à la ligne suivant.

Reply:
Ok.  

Dans ce cas, c'est l'action de **mettre en minuscule** qui est important.  
Je propose.

`Mettre en minuscule le type de fréquence d'envoi dans les résumés.`

Reply:
Pour ma part je propose :

> Fréquence d’envoi (en lettres minuscules) pour le titre du résumé

Reply:
Je ne suis pas ok avec cette proposition. Car tu minimises l'action de mettre en minuscule. Alors que cette option ne sert qu'à cela.
voir ligne de code : https://github.com/MarkDHamill/digests/blob/master/cron/task/digests.php#L679

Reply:
Je plaçais le fond en priorité (fréquence d'envoi) et la forme entre parenthèses (minuscules) mais c'est comme tu veux.
Je n'aime pas "mettre" et tu omets la notion de "titre du résumé" (objet du courriel) qui me semble essentielle pour savoir de quoi on parle (ok l'auteur aussi) .
Je propose : `Saisissez en minuscules la fréquence d’envoi pour le titre du résumé`

Reply:
Il ne s'agit pas d'une description, donc le verbe doit être à l'infinitif.  
Ensuite, on ne peut pas utiliser Saisissez, vu qu'il n'est rien saisie ou qu'il n'y a rien à saisir.

Cette option ne laisse le choix qu'entre "Oui" ou "Non".  
L'action de cette option est de transformer les :
-  `Quotidien` en `quotidien`
-  `Hebdomadaire` en `hebdomadaire`
-  `Mensuel` en `mensuel`

Donc, à moins que tu ne trouves un autre terme en lieu et place de `Mettre`… voici la proposition de trad:

`Mettre en minuscules la fréquence d’envoi dans le titre du résumé`